src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -505,13 +505,6 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
         return this.localeConversionService.toLocaleString(numberToLocalize, this.course.accuracyOfScores);
     }
 
-    /**
-     * Localizes a percent number, e.g. switching the decimal separator
-     */
-    localizePercent(numberToLocalize: number): string {
-        return this.localeConversionService.toLocalePercentageString(numberToLocalize, this.course.accuracyOfScores);
```

Comment:
Before the localizer also performed the rounding of the scores to the correct accuracy as defined by the course. Where does this happen now?

Reply:
Yeah, I totally missed that. I will adapt the code.